### PR TITLE
Chore: merge with repo abcxyz/jvs-plugin-github-dev

### PR DESCRIPTION
**Pull request by sailorlqh**
Merge with repo repo abcxyz/jvs-plugin-github-dev

### CODE REVIEW

File: .github/CODEOWNERS
Comment (verbanicm) [inline .github/CODEOWNERS:1]: Can you add go readability to this?

Comment (sailorlqh) [reply .github/CODEOWNERS:1]: Added

Comment (verbanicm) [reply .github/CODEOWNERS:1]: @sailorlqh  See https://github.com/abcxyz/guardian/blob/main/.github/CODEOWNERS

Comment (sailorlqh) [reply .github/CODEOWNERS:1]: Thanks! Updated.

**Review by verbanicm — APPROVED** [review]

**Pull request merged**